cylc/flow/config.py

```diff
@@ -1618,10 +1615,6 @@ def generate_taskdefs(self, orig_expr, left_nodes, right, seq):
             for item in self.cfg['runtime'][name]['outputs'].items():
                 if item not in taskdef.outputs:
```

Comment:
@trwhitcomb, when you change the following to a set operation,  you won't need the `if` statement after this change.

Reply:
Could you clarify that statement @dwsutherland ?  ("won't need the if statement after this change" ... do you mean the if state *in* this change, or the one *before* this change?  - if the latter, maybe you didn't see this comment: https://github.com/cylc/cylc-flow/pull/3259#discussion_r310430133 )

Reply:
@trwhitcomb can safely ignore this change, in his PR.

Reply:
@hjoliver : https://github.com/cylc/cylc-flow/pull/3268/files#diff-6f57d0a14add56d573847aef0ac65984

Reply:
Yes, that's the one.

Reply:
(understood now!)